bouncer/src/repo/core/model/bson/repo_bson_factory.cpp

```diff
@@ -893,37 +893,34 @@ TextureNode RepoBSONFactory::makeTextureNode(
 	const uint32_t    &byteCount,
 	const uint32_t    &width,
 	const uint32_t    &height,
-	const int         &apiLevel)
+	const int         &apiLevel,
+	const std::vector<repo::lib::RepoUUID>* parentIDs)
```

Comment:
This doesn't need to be a pointer? make it a ref instead would be better i think (or at least inline with everything else)
```suggestion
	const std::vector<repo::lib::RepoUUID> &parentIDs)
```